_team.qmd

```diff
@@ -1,14 +1,33 @@
-## Research Computing Team {.smaller}
-
-Staff: 
-
-- {{< var rc.team >}}
-
-Here for:
-
-- General guidance and advice
-- Access to specialist support and consultancy
-- Research Software Engineering support
-
-Contact via IT Service Desk: {{< var rc.servicedesk >}}
+## Research Computing at Leeds {.smaller}
+<!--
+  - {{< var rc.team >}}
+-->
+- A centralised [Team](https://arc.leeds.ac.uk/about/team/) and service supporting researchers with research computing platforms, consultancy and training.
+- We have capacity to help with developing funding proposals, recruiting specialists and can provide costed support to deliver research projects.
+- For details please see our [Website]({{< var rc.website >}})
+- For support and queries, please [send us a ticket via the IT Service Desk]({{< var rc.servicedesk >}})
 
+::: {.notes}
```

Comment:
If we're having speaker notes, I think we need to decide what their purpose is.  These are lengthy things I need to read ahead of giving the presentation, rather than useful aide memoires to help me whilst I'm presenting.  I have to scroll some of them when I render this at normal size on a 1080p monitor.

Reply:
Yes, you wouldn't need them. I think they might be helpful for others. Perhaps more so in preparation for leading the training than when actually delivering the training...

Reply:
I think I'd prefer a split.  Speaker notes should be helpful whilst you're delivering, whereas it's nice to have more information that's useful for before you start.  To an extent rc-lesson-plans was all about that second part, although it could be thickened up quite a lot.  I'd just be concerned that lots of info in the speaker notes ends up being quite hard work live, as you're not sure what's useful now, and what would have been useful earlier.